managed/src/main/java/com/yugabyte/yw/controllers/CloudProviderController.java

```diff
@@ -213,7 +209,8 @@ public Result create(UUID customerUUID) throws IOException {
     } catch (RuntimeException e) {
```

Comment:
Fix RuntimeException catching business

Reply:
Created another PR to resolve merge conflicts and this review comments 
https://github.com/yugabyte/yugabyte-db/pull/8472

Reply:
Done.

new PR: https://github.com/yugabyte/yugabyte-db/pull/8472